app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -1,5 +1,11 @@
-<p>Welcome <%= @email %>!</p>
+<p>
+	<strong> Welcome <%= @email %>! </strong>
+</p>
 
 <p>You can confirm your account email through the link below:</p>
 
-<p><%= link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token) %></p>
+<p>
+	<%= link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token) 
+	<%= link_to @user.email
+		<button type="button" class="btn btn-primary">email</button>%>
```

Comment:
It looks like the `%>` tag at the end of line 10 needs to be at the end of line 9 instead.  It looks like this is causing an error when a user tries to create a new account.